t/t2201-job-cmd.t

```diff
@@ -767,6 +767,13 @@ test_expect_success 'flux job: timeleft returns 0 for completed job' '
 	flux job timeleft $id > timeleft7 &&
 	grep "0s" timeleft7
 '
+test_expect_success 'flux job: timeleft fails for pending job' '
+	flux queue stop &&
+	id=$(flux mini submit -t 10m true) &&
+	test_expect_code 1 flux job timeleft $id > timeleft8 2>&1 &&
+	flux queue start &&
+	grep "has not started" timeleft8
+'
```

Comment:
this test is in the ` testsuite: add tests for flux.job.timeleft() ` commit.  accidentally put here and not in the other commit with all of the other tests for `t2201-job-cmd`?

Reply:
Ooh, good catch! Thanks, will fix